src/OrchardCore/OrchardCore.ContentManagement.Abstractions/Handlers/ContentHandlerBase.cs

```diff
@@ -1,30 +1,145 @@
-﻿namespace OrchardCore.ContentManagement.Handlers
+using System.Threading.Tasks;
+
+namespace OrchardCore.ContentManagement.Handlers
 {
     public abstract class ContentHandlerBase : IContentHandler
     {
         public virtual void Activating(ActivatingContentContext context) { }
```

Comment:
I would mark the sync ones as obsolete and ensure we are not using them in our code. Then in RC or before we can just remove them. Or just remove them.

Reply:
I'm happy to remove them, but wouldn't it be convenient for sub-classes that don't need async code to use the non-async methods? The interface itself has async methods only of course.